dynamic/main.py

```diff
@@ -73,4 +77,4 @@ def catch_all(path):
 
 
 if __name__ == "__main__":
-    app.run(debug=True, host='0.0.0.0', port=8001)
+    app.run(debug=True, host='0.0.0.0', port=int(os.getenv("PORT", 8001)))
```

Comment:
Are you sure you want debug enabled if we're pushing this to a public paas instance?

Reply:
I'll change it to default False.